sep490-frontend/src/app/modules/admin/components/create-update-package-credit/create-update-package-credit.component.ts

```
@@ -0,0 +1,84 @@
+    private readonly router: Router,
+    private readonly activatedRoute: ActivatedRoute
+  ) {
+    super(httpClient, formBuilder, notificationService, translate);
+  }
+
+  get isEdit(): boolean {
+    return !!this.packageCreditStructure.id.value;
+  }
+  override reset(): void {
+    if (this.isEdit) {
+      return this.initializeData();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)